python-codes/plot-func.py

```diff
@@ -0,0 +1,10 @@
+import numpy as np
+import matplotlib.pylab as plt
```

Comment:
```suggestion
import matplotlib.pyplot as plt
```
`pylab` is the old way.

Reply:
@dutta-alankar I have made the correction. This can now be merged.